reconcile/openshift_resources_base.py

```diff
@@ -596,6 +602,48 @@ def fetch_current_state(
         )
 
 
+def _post_process_route(
+    oc: OCClient,
+    namespace: str,
+    resource: Mapping[str, Any],
```

Comment:
is `resource` required here?

Reply:
wdyt about keeping them in the method signature but using _ as var name to indicate that they are currently not used?

Reply:
consider this a nit and ignore if you like